deploy.Jenkinsfile

```diff
@@ -135,6 +134,36 @@ pipeline {
         }
       }
     }
+
+    stage('Push-Docs') {
+      // Assuming windows
+      steps {
+
+        withCredentials([string(credentialsId: 'GitHub_API_Token',
+                                variable: 'api_token')]) {
+          powershell """
+            git config --local user.name "PACE CI Build Agent"
+            git config --local user.email "pace.builder.stfc@gmail.com"
+            git clone "https://github.com/pace-neutrons/Horace.git" --branch gh-pages --single-branch docs
+            cd docs
+            git remote set-url --push origin "https://pace-builder:\$(\${env:api_token}.trim())@github.com/pace-neutrons/Horace"
+
+            git rm -rf --ignore-unmatch ./${version_number}
+
+            New-Item -Path ./${version_number} -ItemType Directory
+            Expand-Archive -Path ../docs.zip -DestinationPath ./${version_number}
+
+            git add ./${version_number}
+            Write-Host '<meta http-equiv="Refresh" content="0; url=''https://pace-neutrons.github.io/Horace/${version_number}/''" />'
+            Set-Content -Path ./stable/index.html -Value '<meta http-equiv="Refresh" content="0; url=''https://pace-neutrons.github.io/Horace/${version_number}/''" />'
```

Comment:
Could we define longer strings like this in a variable at the top of the script? Especially if we're going to use them more than once.

Similar for the Git username/email/remote.